pages/linux/qm-guest-exec.md

```diff
@@ -0,0 +1,20 @@
+# qm guest exec
+
+> Execute a specific command via a guest agent.
+> More information: <https://pve.proxmox.com/pve-docs/qm.1.html>.
+
+- Execute a specific command via a guest agent:
+
+`qm guest exec {{vm_id}} {{arg1 arg2 ...}}`
```

Comment:
Should the command be placed here?
```suggestion
`qm guest exec {{vm_id}} {{command}} {{arg1 arg2 ...}}`
```

Reply:
Ref: `qm guest exec <vmid> [<extra-args>] [OPTIONS]`

So shouldn't it be arguments first then commands (Options)? Also, we should update the other commands right.